app/Http/Controllers/api/TagController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Http\Controllers\api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\TagRequest;
+use App\Http\Resources\TagResource;
+use App\Models\Tag;
+use Illuminate\Http\Exceptions\HttpResponseException;
+
+class TagController extends Controller
+{
+    public function index()
+    {
+        $tags = Tag::get();
+
+        if ($tags->isEmpty()) {
+            throw new HttpResponseException(response()->json(['message' => __('No hi ha tags a la base de dades')], 404));
+        }
```

Comment:
This key is not at the language files, either on `lang\ca\http-statuses.php` or on `lang\ca.json`. I'm not sure if my suggestion it´s ok.
```suggestion
            throw new HttpResponseException(response()->json(['message' => __('Not Found')], 404));
```
Same for the rest of the outputs.

Reply:
I still don't get in which language should we be throwing answers and execptions, i think would be a good idea to discuss as a group with the whole IT profiles collabs/mentors

Reply:
As we discussed before, I am making that change. tnxs for the insight!